Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift

```diff
@@ -135,7 +135,7 @@ struct AnimationsView: View {
           "Big mode",
           isOn:
             viewStore
-            .binding(get: \.isCircleScaled, send: Animations.Action.circleScaleToggleChanged)
+            .binding(get: \.isCircleScaled, send: { .circleScaleToggleChanged($0) })
```

Comment:
**Question** 
Why do you use `binding(get:send:)` instead of using `BindingState`?
```swift
viewStore.$isCircleScaled
```
Is this to avoid changing the logic since you've already provided the episode video?
*OR* is there any case where it *prefers to* use `binding(get:send:)`?